libdnf/rpm/package_set_impl.hpp

```diff
@@ -62,8 +62,8 @@ inline PackageSet::Impl::Impl(SolvSack * sack)
     , sack(sack->get_weak_ptr()) {}
 
 
-inline PackageSet::Impl::Impl(SolvSack * sack, Map * map)
-    : libdnf::rpm::solv::SolvMap::SolvMap(map)
+inline PackageSet::Impl::Impl(SolvSack * sack, libdnf::rpm::solv::SolvMap & solv_map)
+    : libdnf::rpm::solv::SolvMap::SolvMap(solv_map)
```

Comment:
We are in libdnf::namespace:
`libdnf::rpm::solv::SolvMap::SolvMap(solv_map)` -> `solv::SolvMap::SolvMap(solv_map)`

Reply:
> Commit messages:
> Add ifilter for SolvQuery -> ifilter for SolvQuery + format (clang) changes
> Rename SolvQuery.ifilter_nevra_strict : sorter -> shorter

Fixed